### PR TITLE
feature: update python 3.8

### PR DESCRIPTION
**Pull request by ahsan-z-khan**
*Issue #, if available:*

*Description of changes:*
Updated Buildspecs to support python 3.8 
Updated ECR image to 20.04

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def build_extension(self, ext):
 
 setup(
     name='sagemaker_tensorflow',
-    version='2.6.0.2.0.0',
+    version='2.6.0.1.10.0',
```
Comment (satishpasumarthi) [inline setup.py]: is this version change correct?

Comment (ahsan-z-khan) [reply setup.py]: seems like it will be 1.11.0

**Review by satishpasumarthi — DISMISSED** [review]
LGTM

**Review by shreyapandit — DISMISSED** [review]

File: setup.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def build_extension(self, ext):
 
 setup(
     name='sagemaker_tensorflow',
-    version='2.6.0.2.0.0',
+    version='2.6.0.1.11.0',
```
Comment (saimidu) [inline setup.py:102]: Why is this "downgraded"?

Comment (shreyapandit) [reply setup.py:102]: My exact question. Looks like the earlier version bump to 2x happened mistakenly and this is the correct ordering. cc @ahsan-z-khan

Comment (ahsan-z-khan) [reply setup.py:102]: yes +1 to what @shreyapandit  said

**Review by saimidu — DISMISSED** [review]

**Review by shreyapandit — APPROVED** [review]

**Review by satishpasumarthi — APPROVED** [review]
looks good

**Review by saimidu — COMMENTED** [review]
Approved.

**Pull request merged**